modules/vba_documents/config/routes.rb

```diff
@@ -2,12 +2,9 @@
 
 VBADocuments::Engine.routes.draw do
   get '/metadata', to: 'metadata#index'
-  get '/v0/healthcheck', to: 'metadata#healthcheck'
+  match '/v1/*path', to: 'application#cors_preflight', via: [:options]
```

Comment:
For all other APIs, the CORS preflight route is listed before the other versioned routes, which is why I changed the route order here.